DogeNet.BLL/Features/Account/AccountChecks.cs

```diff
@@ -11,14 +11,12 @@ public static class AccountChecks
     {
         public static bool UniqueName(DBContext db, string name)
         {
-            if (db.Users.Where(o => o.UserName == name).Count() == 0)
-            {
-                return true;
-            }
-            else
-            {
-                return false;
-            }
+            return db.Users.Where(o => o.UserName == name).Count() == 0;
```

Comment:
What about using Any()? For me, it seems better. It is just a recommendation)

Reply:
Done
![image](https://user-images.githubusercontent.com/77879337/145008316-8b68add3-6c1e-4832-a60a-dde27a6c2b3e.png)